tests/test_integrated_channels/test_sap_success_factors/test_exporters/test_course_metadata.py

```diff
@@ -67,3 +67,39 @@ def test_unparsable_language_code(self):
         assert str(exc_info.value) == (
             'Language codes may only have up to two components. Could not parse: this-is-incomprehensible'
         )
+
+    @responses.activate
+    def test_transform_title_includes_start(self):
+        """
+        Transforming a title gives back the title and start date if the course is instructor-paced.
+        """
+        course_run = {
+            'start': '2013-02-05T05:00:00Z',
+            'pacing_type': 'instructor_paced',
+            'title': 'edX Demonstration Course'
+        }
+        exporter = SapSuccessFactorsCourseExporter('fake-user', self.config)
+        assert exporter.transform_title(course_run) == \
+               [{
+                   'locale': 'English',
+                   'value': 'edX Demonstration Course (Starts: February 2013)'
+               }]
+
+    @responses.activate
+    def test_transform_title_excludes_start(self):
+        """
+        Transforming a title gives only returns the title (not start date) if the course isn't instructor-paced.
+        """
+        course_run = {
+            'start': '2013-02-05T05:00:00Z',
+            'pacing_type': 'self_paced',
+            'title': 'edX Demonstration Course'
+        }
+        exporter = SapSuccessFactorsCourseExporter('fake-user', self.config)
+        assert exporter.transform_title(course_run) == \
+            [{
+                'locale': 'English',
+                'value': 'edX Demonstration Course'
+            }]
```

Comment:
If you have the title format code in a shared function, you probably only need one set of these tests.